src/screen/KeyList/styles.ts

```diff
@@ -28,37 +33,68 @@ export const HeaderTitle = styled.p`
   line-height: 24px;
 `
 
-export const HeaderTotalKeys = styled.p`
+export const HeaderDatabaseDetails = styled.p`
   font-weight: normal;
   font-size: 12px;
   line-height: 14px;
   color: #9696a7;
   margin-left: 8px;
+
+  span + span {
+    margin-left: 8px;
+    padding-left: 8px;
+    border-left: 1px solid rgba(255, 255, 255, 0.1);
+  }
 `
 
-export const KeyListContainer = styled.section`
+export const KeyListWrapper = styled.div`
+  height: calc(100vh - 140px);
+  overflow: auto;
   margin-top: 18px;
 `
 
+export const KeyListContainer = styled.section`
+  position: relative;
+`
+
 export const Key = styled.div`
   display: flex;
   flex-direction: row;
+  align-items: center;
   justify-content: space-between;
   padding: 8px 0;
 
-  &:not(:last-child) {
-    border-bottom: 1px solid rgba(225, 225, 230, 0.1);
+  & + div {
+    border-top: 1px solid rgba(225, 225, 230, 0.1);
   }
 
-  svg {
-    color: #9696a7;
+  button {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+
+    svg {
+      color: ${props => props.theme.colors.grey};
+      width: 14px;
+      height: 14px;
+    }
+
+    &:hover svg {
+      color: ${props => props.theme.colors.white};
+    }
   }
 `
 
-export const KeyTextContainer = styled.div`
+interface KeyTextContainerProps {
+  selected: boolean
+}
+
+export const KeyTextContainer = styled.button<KeyTextContainerProps>`
   display: flex;
   flex-direction: row;
   align-items: center;
+  color: ${props =>
+    props.selected ? props.theme.colors.pink : props.theme.colors.white};
```

Comment:
Maybe it's good idea create method hover
Example:
```ts
&:hover{
color: props.theme.colors.pink;
opacity:.8;
}
```

Reply:
Fixed.